packages/buckr/src/index.ts

```diff
@@ -0,0 +1,339 @@
+import fs from 'fs'
```

Comment:
did this as an experiment the other day, but figured we might as well capture these methods. if nothing else they serve as good usage examples for typescript.